benchmark/scenarios/__init__.py

```diff
@@ -1 +1,5 @@
 from .no_load import NoLoadScenario
+from .read_request_load import (
+    ReadRequestLoadScenario, RateMeasurer, RequestRateTooLow,
+    RequestRateNotReached, RequestOverload
+)
```

Comment:
Only export truly public objects in `__init__.py`. The `RateMeasurer` is an internal object, and it's only expected use is in the test module.  The test module can import directly from the module being tested.